autosklearn/automl.py

```diff
@@ -1033,7 +1035,6 @@ def fit_ensemble(self, y, task=None, precision=32,
         )
         manager.build_ensemble(self._dask_client)
         future = manager.futures.pop()
-        dask.distributed.wait([future])  # wait for the ensemble process to finish
```

Comment:
Did you remove this because `future.result()` waits by itself?

Reply:
Yes!